BootloaderCorePkg/Tools/GenContainer.py

```diff
@@ -13,6 +13,13 @@
 from   ctypes import *
 from   CommonUtility import *
 
+AUTH_TO_HASHALG_STR = {

```

Comment:
let us move it into CONTAINER() scope like _auth_type_value.

Reply:
Done